styles.css

```diff
@@ -0,0 +1,144 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+a {
+  color: white;
+  font-weight: 100;
+  letter-spacing: 2px;
+  text-decoration: none;
+  display: inline-block;
+  width: 100%;
+  text-align: center;
+  transition: all 0.5s;
+}
+
+.navscontent {
+  padding: 30px 5px;
+}
+
+a:hover {
+  background: gray;
+}
+
+.container {
+  max-width: 1000px;
+  margin: 0 auto;
+  padding: 20px;
+}
```

Comment:
- Kindly already we told do not use width in px always use width percentage so your webpage got responsive 
and also remove padding and margin to remove extra white space around the webpage